src/env/root/mod.rs

```diff
@@ -386,9 +405,22 @@ impl<'a> RootRuntime<'a> {
 
         Ok(Some(retcode.into()))
     }
+
+    #[cfg(feature = "debug")]
+    fn ext_print(&self, args: RuntimeArgs) -> ExtResult {
+        let memory = self.memory();
+
+        let ptr: u32 = args.nth(0);
+        let len: u32 = args.nth(1);
+
+        let bytes = memory.get(ptr, len as usize).unwrap();
+
+        self.print(&bytes);
+
+        Ok(None)
+    }
 }
 
-#[derive(Debug)]
```

Comment:
So by adding the closure for `logger`, I lost the ability to derive the `Debug` trait. Do you think that I should manually derive it so that we can put add the `Debug` trait back?